content/300-accelerate/580-local-development.mdx

```diff
@@ -8,7 +8,7 @@ toc: true
 
 <TopBlock>
 
-Prisma Accelerate is designed to scale database connections efficiently in production environments. Prisma Accelerate also provides a global cache to reduce the load on your database and reduce query response time. To leverage Prisma Accelerate, it is essential to utilize a publicly accessible database.
+Prisma Accelerate is designed to scale database connections efficiently in production environments. Prisma Accelerate also provides a global cache to reduce the load on your database and reduce query response time. To leverage Prisma Accelerate, it is essential to utilize a deployed database.
```

Comment:
What is a non deployed database? Is this sentence still valuable without the "publicly accessible" part?

Reply:
It would be valuable as you'd still need a database that is deployed ( vs. a local database )for you to use Accelerate's caching.

Do you think using an alternative text such as "a hosted database"  would make it more clear?

Reply:
"hosted" makes more sense to me. I'm still fuzzy on what we're wanting to say here, and why: why is a local database not usable? Do a people run their production environments wiht local dbs? What are we trying to say here, that you need a DB that's reachable by Accelerate, or something else?

Reply:
I think that's better, but I'm still unsure if that conveys a meaningful addition and is valuable. Isn't it implied and obvious that to use Accelerate, it needs to be able to connect to your database?

@luanvdw, do you have any input on this maybe?

Reply:
I get the desire to explicitly mention production in order to create the mental connection of the paragraph that follows into development. I think we can simplify it with;

Prisma Accelerate efficiently scales production traffic with advanced connection and a global database cache.

It captures the value in **production** from which you can then start the follow up paragraph with;

During development you may want to use a local database...

To create that separation without explicitly mentioning hosted vs non hosted or reachability.

Reply:
```suggestion
Prisma Accelerate efficiently scales production traffic with advanced connection and a global database cache.
```

Reply:
@ankur-arch let's add the word integrated to connection pooling.

Prisma Accelerate efficiently scales production traffic with integrated connection pooling and a global database cache.